CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.0]
+
+### Added
+
+- Multichain Test Dapp: Add (multi)input to request accounts in `wallet_createSession` call #23 ([#31](https://github.com/MetaMask/test-dapp-multichain/pull/31))
+- Implement Wallet Discovery For Multichain API ([#22](https://github.com/MetaMask/test-dapp-multichain/pull/22))
+
+### Changed
+
+- Move [CAIP-294](https://github.com/ChainAgnostic/CAIPs/blob/bc4942857a8e04593ed92f7dc66653577a1c4435/CAIPs/caip-294.md) events into useSDK ([#30](https://github.com/MetaMask/test-dapp-multichain/pull/30))
```

Comment:
thoughts on adding `([#30](https://github.com/MetaMask/test-dapp-multichain/pull/30))` to the end of line 15 and then deleting this "Changed" entry? Because as far as the consumer of this package is concerned, between v0.3.2 and v0.4.0, the only change here is that Wallet Discovery was added